src/Entities/Player.h

```diff
@@ -682,6 +689,12 @@ class cPlayer :
 
 	AString m_CustomName;
 
+	/** Displayed skin part bit mask */
+	UInt8 m_SkinParts;
```

Comment:
No, `enum` means "one of", you want multiple choice, so it has to be a regular numeric type. Another possibility would be `std::set<eSkinPart>`, but that would be an overkill and it would be difficult to convert between the protocol value and this value. I believe an `unsigned` is the best approach here.